CMakeLists.txt

```diff
@@ -80,6 +80,16 @@ if(BUILD_TESTING)
   add_launch_test(test/two_links_fixed_joint-launch.py
     ARGS "test_exe:=$<TARGET_FILE:test_two_links_fixed_joint>")
 
+  add_executable(test_two_links_fixed_joint_prefix test/test_two_links_fixed_joint_prefix.cpp)
+  ament_target_dependencies(test_two_links_fixed_joint_prefix
+    rclcpp
+    sensor_msgs
```

Comment:
nit, unnecessary dependency?

Reply:
Yes, definitely.  Removed it from here and from `test_two_links_fixed_joint` in e83f247